Sources/Networking/OAuth/OAuthAuthenticator.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  OAuthAuthenticator.swift
+//  Catalog
+//
+//  Created by Zvonimir Medak on 10.10.2022..
+//  Copyright © 2022 Infinum. All rights reserved.
+//
+
+import Alamofire
+import Combine
```

Comment:
Combine not needed, also Foundation can be removed as it's imported inside Alamofire

```suggestion

```